docs/index.html

```
@@ -311,41 +311,39 @@ <h1><span class="thin">A guide for people who are not developers</span>Forge</h1
   <p>
-    What you should notice: finishing a piece of work takes fewer steps and produces fewer
-    contradictions, the summaries are always current because nobody maintains them, and asking
-    "what is left to do" reads a short generated list instead of a very long history. Before
-    anything is pushed, Forge checks that every note still points at something real.
+    And before a release, "the automated checks passed" is no longer something Forge can take
+    from its own notes. It now looks up the result for the exact version of the code being
+    released and quotes it. A release went out over a failing check because that line had been
+    written a day earlier and nobody re-read it. More generally, a recorded check result older
+    than the code it covers now counts as missing rather than as proof.
   </p>
   <p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 343.

(tag-pair)